src/objects/table/table.scss

```diff
@@ -0,0 +1,84 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+
+$table-border-color: colors.$gray-light;
+$table-border-thin: sizes.$edge_small solid $table-border-color;
+$table-border-thick: sizes.$edge_medium solid $table-border-color;
```

Comment:
Honest question I asked myself (but didn't answer) for button: Should these values be stored in a design token file? If so, should that token live with the pattern or inside of tokens?

These are the sorts of questions that make me lie awake at night. 🙃

Reply:
I'm not sure...

To be honest I don't have a ton of experience with design tokens.

From my perspective it would be useful to make these design tokens if we wanted to use them in multiple places. e.g. I want to use them for table styles and document them separately in storybook so future devs can use them in similar interfaces.

At this point I don't see a second use case so it seems like a premature optimization but I'm open to other opinions, especially if folks have more experience with design tokens.